src/qt/pivx/send.cpp

```diff
@@ -394,9 +394,16 @@ void SendWidget::ProcessSend(const QList<SendCoinsRecipient>& recipients, bool h
 {
     // First check SPORK_20 (before unlock)
     bool isShieldedTx = hasShieldedOutput || !isTransparent;
-    if (isShieldedTx && walletModel->isSaplingInMaintenance()) {
-        inform(tr("Sapling Protocol temporarily in maintenance. Shielded transactions disabled (SPORK 20)"));
-        return;
+    if (isShieldedTx) {
+        if (!walletModel->isSaplingEnforced()) {
+            inform(tr("Cannot perform shielded operations, v5 upgrade hasn't being enforced yet!"));
```

Comment:
change to either `hasn't been enforced` or `isn't being enforced`

Reply:
typo fixed